atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraServer.java

```diff
@@ -41,6 +42,7 @@ public interface CassandraServer {
     /**
      * The only proxy that will be used to reach the Cassandra host.
      * */
+    @DoNotLog
```

Comment:
Any reason why this is not `DoNotLog`, but ` reachableProxyIps` does not have such annotation?

Reply:
good question! I think this is because of the redaction, see response to https://github.com/palantir/atlasdb/pull/6279/files/1cbb89e962239fbde18e749605d4736b4be4f393#r992426287